### DISCUSSION TITLE
Query Error: Request to ... failed, reason: unable to verify the first certificate

### DISCUSSION THREAD

**Discussion by simoncramer** [Budibase/budibase#8808] [Help]
Hi!
I am trying to connect to my REST API, but I am getting the error:

`Query Error: request to ... failed, reason: unable to verify the first certificate`

Is there anything I can do about this? In my browser and when using python code the API works fine.
I have deployed budibase via your Helm chart.

Thanks!

**Comment by jonnymccullagh** [comment]
Hi Simon, I have just tested our QA helm installation with a Chuck Norris API endpoint and it succeeds. 
![image](https://user-images.githubusercontent.com/5209315/204022890-cfbc7042-c334-4cd2-b145-0b357183b6e2.png)
If you would like us to test against your API endpoint feel free to DM any of us on Discord.

**Comment by simoncramer** [comment]
Hi @jonnymccullagh,
the problem seems to be that my API is delivering an incomplete SSL certificate where intermediate certificates are missing.
Hence some nodeJS library is not communicating correctly. Unfortunately, I cannot change the ingress config of my API.

I fixed this for my installation in the following way: 
1. Call `helm install --debug --dry-run --namespace budibase budibase budibase/budibase > budibase.yaml`
2. Extract the deployments and services from the YAML file.
3. Add the following environment variable to every deployment I could find:
```
- name: NODE_TLS_REJECT_UNAUTHORIZED
  value: "0"
```
4. Deploy manually.

Maybe you can add a flag `tls-reject-unauthorized` to your helm values, so that others can set this environment variable more easily?
Thanks!
